apps/ledger-live-mobile/src/screens/Portfolio/useListsAnimation.ts

```diff
@@ -3,12 +3,15 @@ import { useState, useEffect } from "react";
 import { TAB_OPTIONS } from "./TabSection";
 import { track } from "~/analytics";
 import { LayoutChangeEvent } from "react-native/Libraries/Types/CoreEventTypes";
+import { useDispatch } from "react-redux";
+import { setSelectedTabPortfolioAssets } from "~/actions/settings";
 
-type TabListType = (typeof TAB_OPTIONS)[keyof typeof TAB_OPTIONS];
+export type TabListType = (typeof TAB_OPTIONS)[keyof typeof TAB_OPTIONS];
```

Comment:
Imho, you should convert TAB_OPTIONS as an enum, and then use it everywhere, you will not have to cast and it's better to understand

Reply:
Then you can use it in your store and also your component

it's type safe